### PR TITLE
[ISSUE-176] support upgrade for loopbackmgr

### LINKED ISSUE #176: Loopbackmgr don't  handle CSI upgrade properly
After upgrade with helm:
```
helm upgrade csi-baremetal charts/baremetal-csi-plugin --reuse-values --set image.tag=`make version`
```
New devices were created, old one reported as OFFLINE:

```
Id                                     Size        Type   SN                   Status    Health    OpStatus    Node
da4c9bb7-9145-4474-a716-be2d8b0b07eb   104857600   HDD    LOOPBACK2469845418   ONLINE    GOOD      OPERATIVE   2aaaa75f-5649-4a7a-be26-d9c07abfed97
11bb7d25-cb40-4d43-9833-671a9be9a7f5   104857600   HDD    LOOPBACK153357917    OFFLINE   UNKNOWN   RELEASING   2aaaa75f-5649-4a7a-be26-d9c07abfed97
15ec4ff0-5d35-45ca-944b-23cf0ca6e70c   104857600   HDD    LOOPBACK1341594720   ONLINE    GOOD      OPERATIVE   2aaaa75f-5649-4a7a-be26-d9c07abfed97
68b4ca64-dfca-4bb2-a12c-e707cceda078   104857600   HDD    LOOPBACK3740241886   OFFLINE   UNKNOWN   OPERATIVE   2aaaa75f-5649-4a7a-be26-d9c07abfed97
f7446fe3-abf8-4bda-8b8a-55d72ee600d0   104857600   HDD    LOOPBACK556698644    OFFLINE   UNKNOWN   OPERATIVE   2aaaa75f-5649-4a7a-be26-d9c07abfed97
5e554a26-1983-4856-b2e6-6cd028fa33ae   104857600   HDD    LOOPBACK2374520789   ONLINE    GOOD      OPERATIVE   2aaaa75f-5649-4a7a-be26-d9c07abfed97
```

Need to rely on bm-csi nodes CRD

### PR DESCRIPTION
**Pull request by painhardcore**
## Purpose
Upgrading with helm creates new devices and leave the old ones as OFFLINE.
These changes will use a unique nodeID from the annotation which CSIBM operator creates.
Also, changed sizeOfLoopDevices as an old tail from #56 

## PR checklist
- [x] Add link to the issue
- [x] Choose PR label
- [x] New unit tests added
- [x] Modified code has meaningful comments
- [ ] All TODOs are linked with the issues
- [ ] All comments are resolved
- [ ] PR validation is passed
- [x] Custom CI is passed

## Testing
Manual upgrade

fixes #176

### CODE REVIEW

File: charts/baremetal-csi-plugin/templates/baremetal-csi-node.yaml [hunk 1]
```diff
@@ -150,6 +150,8 @@ spec:
         args:
           - --loglevel={{ .Values.log.level }}
           - --drivemgrendpoint={{ .Values.drivemgr.grpc.server.endpoint }}
+          - --usenodeannotation={{ .Values.feature.usenodeannotation }}
```
Comment (mishoyama) [inline charts/baremetal-csi-plugin/templates/baremetal-csi-node.yaml:153]: These changes are only needed for `loopback manager`. Can you make them `loopback manager` specific only? Can we also read `node name` through environment variable?

**Review by mishoyama — CHANGES_REQUESTED** [review]

File: charts/baremetal-csi-plugin/templates/baremetal-csi-node.yaml (hunk 1 above)
Comment (painhardcore) [reply charts/baremetal-csi-plugin/templates/baremetal-csi-node.yaml:153]: @mishoyama 
Yep, sure.
Do we have a rule about things we pass as parameters and others with env? `nodename` is already passed as a parameter in the node container, so why we should use a different approach in drivemgr?

Comment (mishoyama) [reply charts/baremetal-csi-plugin/templates/baremetal-csi-node.yaml:154]: No, we don't have such rule. But if we can discover something just using standard k8s env vars why do we need to have extra argument?

**Review by mishoyama — APPROVED** [review]

**Review by ykulazhenkov — APPROVED** [review]

File: charts/baremetal-csi-plugin/templates/baremetal-csi-node.yaml (hunk 1 above)
Comment (painhardcore) [reply charts/baremetal-csi-plugin/templates/baremetal-csi-node.yaml:154]: I agree, but `nodename` is KUBE_NODE_NAME env and it's not the standard k8s env. It's made from spec.
Should I change the node container to use env instead?

**Pull request merged**